packages/console/src/components/InlineUpsell/index.tsx

```diff
@@ -1,37 +1,39 @@
+import { type AdminConsoleKey } from '@logto/phrases';
 import classNames from 'classnames';
 import { type TFuncKey } from 'i18next';
 import { Trans, useTranslation } from 'react-i18next';
 
-import { contactEmailLink } from '@/consts';
 import InlineNotification from '@/ds-components/InlineNotification';
-import TextLink from '@/ds-components/TextLink';
 import useTenantPathname from '@/hooks/use-tenant-pathname';
 
+import ContactUsPhraseLink from '../ContactUsPhraseLink';
+
 import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
   for: TFuncKey<'translation', 'admin_console.upsell.paywall'>;
+  actionText?: AdminConsoleKey;
```

Comment:
better to add TSDoc comment for this prop. should be clear and show the default value

Reply:
Updated.
<img width="1024" alt="image" src="https://github.com/logto-io/logto/assets/10806653/8e59e2de-33bc-4067-9fa5-92bfa6e4f36f">

Reply:
"Optional {@link AdminConsoleKey} for the action text" exactly equals to `actionText?: AdminConsoleKey`?

Reply:
it's more about what is action text? e.g. "The text to be shown for ..."

Reply:
hey bro you can google for the syntax

Reply:
Updated.